staging_docs/user/tutorials/quickstart.md

```diff
@@ -1,47 +1,34 @@
 # User Setup
 
-All workflow examples use the Pulp CLI. Install and setup from PyPI:
+### Server Installation
```

Comment:
We've discussed about using `oci-images` as the main source of installation info.
So I think it is enough to provide their quickstart, which is already in the staging-site [here](https://staging-docs.pulpproject.org/pulp-oci-images/docs/admin/tutorials/quickstart/).

To add an internal link for `https://staging-docs.pulpproject.org/pulp-oci-images/docs/admin/tutorials/quickstart/` you should use a pattern of `[name](site:url-path.md)`:

```
[Installation Quickstart](site:pulp-oci-images/docs/admin/tutorials/quickstart.md)
```